docs/faq.md

```diff
@@ -24,16 +28,34 @@ For Windows PowerShell:
 $(Split-Path -Parent (Split-Path -Parent (Get-Command griptape-nodes | Select-Object -ExpandProperty Source)))
 ```
 
-## How do I uninstall?
+<a id="uninstall"></a>
+
+## How do I uninstall Griptape Nodes?
 
 Need to part ways with Griptape Nodes? It's a simple goodbye with a single command:
 
 ```bash
-griptape-nodes uninstall
+uv tool uninstall griptape-nodes
 ```
 
 When regret inevitably washes over you, have no fear. Open arms await; just revisit [Getting Started](getting_started.md)
 
+## How do I update Griptape Nodes?
+
+Griptape Nodes will automatically check if it needs to update every time it runs. If it does, you will be prompted to answer with a (y/n) response. Respond with a y and it will automatically update to the latest version of the Engine. Once completed, simply type griptape-nodes or gtn to launch again.
+
+If you would like to _manually_ update, you can always use either of these commands:
+
+`griptape-nodes update`
```

Comment:
total nit - if you can use the same technique here of using the 

```bash
griptape-nodes update
```

so the user can copy and paste the code, that'd be awesome. :) Thanks!